src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -110,66 +118,101 @@ const WorkflowDashboard = _.flow(
     }), simplifiedFailures)
   }
 
+  const callNames = _.sortedUniq(_.keys(calls))
```

Comment:
`sortedUniq` isn't what we want here, try:
```suggestion
  const callNames = _.sortBy(_.identity, _.keys(calls))
```